routes/publishedSchedule.js

```diff
@@ -0,0 +1,150 @@
+const express = require('express');
+const { db } = require('../server/db');
+const { keysToCamel } = require('../common/utils');
+
+const publishedScheduleRouter = express.Router();
+
+// GET - Returns all data from the published_schedule table
+publishedScheduleRouter.get('/', async (req, res) => {
+  try {
+    const allPublishedSchedules = await db.query(
+      `
+      SELECT
+        PS.id,
+        C.host,
+        C.title,
+        PS.confirmed,
+        PS.confirmed_on,
+        PS.start_time,
+        PS.end_time,
+        PS.cohort,
+        PS.notes
+      FROM
+        published_schedule PS
+        LEFT JOIN catalog C ON PS.event_id = C.id;
+    `,
+    );
+    res.status(200).json(keysToCamel(allPublishedSchedules));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+// GET/:id - returns the rows that match the given id
+publishedScheduleRouter.get('/:id', async (req, res) => {
+  try {
+    const publishedScheduleResult = await db.query(
+      `
+      SELECT
+        PS.id,
+        C.host,
+        C.title,
+        PS.confirmed,
+        PS.confirmed_on,
+        PS.start_time,
+        PS.end_time,
+        PS.cohort,
+        PS.notes
+      FROM
+        published_schedule PS
+        LEFT JOIN catalog C ON PS.event_id = C.id
+        AND PS.id = $1;
+      `,
+      [req.params.id],
+    );
+    res.status(200).json(keysToCamel(publishedScheduleResult));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+// POST - Adds a new row to the published_schedule table
+publishedScheduleRouter.post('/', async (req, res) => {
+  const {
+    id,
+    event_id: eventId,
```

Comment:
Can we change the variables from underscores to camelCase here and in the other requests? When it gets called in the front-end it'll already be camelCase.